contracts/ETHAssetHolder.sol

```diff
@@ -0,0 +1,91 @@
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+import './Outcome.sol';
+import './AssetHolder.sol';
+contract ETHAssetHolder is AssetHolder {
+    address AdjudicatorAddress;
+
+    constructor(address _AdjudicatorAddress) public {
+        AdjudicatorAddress = _AdjudicatorAddress;
+    }
+
+    modifier AdjudicatorOnly {
+        require(msg.sender == AdjudicatorAddress, 'Only the NitroAdjudicator is authorized');
+        _;
+    }
+
+    // **************
+    // ETH and Token Management
```

Comment:
Just "ETH Management"? Or do we even need this?

Reply:
👍

Reply:
Addressed in 9b1fbc6